app/views/fruits/show.html.erb

```diff
@@ -3,17 +3,31 @@
   <h3><%= @fruit.sci_name %></h3>
   <p><%= @fruit.description %></p>
 
+  <h1>Submit Your Own Review</h1>
+
+<% if current_user %>
```

Comment:
if a non-logged in user sees this page they will see `Submit Your Own Review` with no form. Consider moving that text within your if

Reply:
good call